### ISSUE TITLE
rmio package not found on CRAN

### ISSUE THREAD

**Issue by hlapp** [NESCent/popgen-docker#29]
Container builds for master are currently failing with this error:
```
Error: (converted from warning) package ‘rmio’ is not available (for R version 3.5.2)
```

It seems that this is a dependency from pcadapt, but looking at its DESCRIPTION it must be indirect. Not sure where this might be resolved by moving pcadapt back to being installed from Github (see #23).

**Comment by hlapp** [comment]
Actually, the dependency [is indeed there](https://github.com/bcm-uga/pcadapt/blob/master/DESCRIPTION#L26), and there is no `Remotes:` repo for the dependency. cc @keurcien

So it seems it's indeed best to switch pcadapt back to installing from CRAN.

**Comment by hlapp** [comment]
Addressed with #30. Also released subsequently as v0.2.12.
